build.gradle

```diff
@@ -47,14 +47,16 @@ configurations {
 
 dependencyManagement {
     imports {
-        mavenBom 'uk.gov.hmcts.reform.idam:idam-bom:1.4.3'
+        mavenBom 'uk.gov.hmcts.reform.idam:idam-bom:1.4.10'
     }
 }
 
 dependencies {
-    generatedCompile group: 'org.springframework', name: 'spring-web'
-    generatedCompile group: 'javax.validation', name: 'validation-api'
-    generatedCompile group: 'io.springfox', name: 'springfox-swagger2'
+    generatedCompile group: 'org.springframework', name: 'spring-web', version: '5.1.5.RELEASE'
+    generatedCompile group: 'javax.validation', name: 'validation-api', version: '2.0.1.Final'
+    generatedCompile group: 'io.springfox', name: 'springfox-swagger2', version: '2.9.2'
+    generatedCompile group: 'com.fasterxml.jackson.core', name: 'jackson-databind', version: '2.9.8'
```

Comment:
we must use a different dependency than apache tomcat embed as this is what Spring Boot uses to start up. it will almost certainly cause some problems for us eventually